terraform/README.md

```diff
@@ -0,0 +1,37 @@
+# Terraform Deployment
+
+This directory contains terraform-driven deployments for PanelApp.
+
+### Prerequisites
+
+1. Create AWS S3 bucket and DynamoDB table for terraform remote state and locking. Refer `backend.conf.sample` for sample backend config.
+    - Enable `Versioning` for S3 bucket
+    - Use `LockID` as Primary key, String type for DynamoDB table
+    - Then, initialize terraform with `-backend-config` flag:
+    ```
+    cp backend.conf.sample backend.conf
+    terraform init -backend-config="backend.conf"
+    ```
+    
+2. Create the following environment specific secret parameters in AWS Parameter Store. If you are planning for more than one environment, replicate more of these parameters with respective environment shorthand variables: `env`, `staging`, `prod`
```

Comment:
I think 'dev' was meant here rather than 'env'...

Reply:
Thanks Andrew. I have updated it.